puppet/zulip/files/nagios_plugins/zulip_postgresql/check_fts_update_log

```diff
@@ -28,7 +31,41 @@ def report(state: str, num: str) -> None:
     exit(states[state])
 
 
-conn = psycopg2.connect(database="zulip")
+pg_args = {}
+
+# Path to the root of the Zulip codebase in production
+sys.path.insert(0, "/home/zulip/deployments/current")
+# Path to the root of the Zulip codebase in development
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../..")))
+os.environ["DJANGO_SETTINGS_MODULE"] = "zproject.settings"
+from django.conf import settings
+
+if settings.REMOTE_POSTGRES_HOST != "":
+    pg_args["host"] = settings.REMOTE_POSTGRES_HOST
+if settings.REMOTE_POSTGRES_PORT != "":
+    pg_args["port"] = settings.REMOTE_POSTGRES_PORT
+
+if "host" in pg_args:
+    pg_args["password"] = ""
+    if settings.DATABASES["default"]["PASSWORD"] is not None:
+        pg_args["password"] = settings.DATABASES["default"]["PASSWORD"]
+    pg_args["user"] = settings.DATABASES["default"]["USER"]
+    pg_args["dbname"] = settings.DATABASES["default"]["NAME"]
+    if settings.REMOTE_POSTGRES_SSLMODE != "":
+        pg_args["sslmode"] = settings.REMOTE_POSTGRES_SSLMODE
+    else:
+        pg_args["sslmode"] = "verify-full"
+    pg_args["connect_timeout"] = "600"
+elif get_config(get_config_file(), "postgresql", "database_name") != "zulip":
+    pg_args["dbname"] = settings.DATABASES["default"]["NAME"]
+    if settings.DATABASES["default"]["USER"] != "zulip":
+        pg_args["user"] = settings.DATABASES["default"]["USER"]
+        pg_args["password"] = settings.DATABASES["default"]["PASSWORD"]
+        pg_args["host"] = settings.DATABASES["default"]["HOST"]
+else:
+    pg_args["user"] = "zulip"
```

Comment:
This is a complex tangle of conditionals that seems to duplicate logic in the `settings.py`; system we should figure out some way to not duplicate this.  Likely just accessing `from django.db import connection` and then `connection.cursor` will work.